testing/scenario_app/ios/Scenarios/ScenariosUITests/PlatformViewUITests.m

```diff
@@ -30,13 +30,18 @@ - (void)testPlatformView {
   NSString* path = [bundle pathForResource:goldenName ofType:@"png"];
   UIImage* golden = [[UIImage alloc] initWithContentsOfFile:path];
 
-  XCUIScreenshot* screenshot = [[XCUIScreen mainScreen] screenshot];
   if (golden) {
     XCTAttachment* goldenAttachment = [XCTAttachment attachmentWithImage:golden];
     goldenAttachment.lifetime = XCTAttachmentLifetimeKeepAlways;
     [self addAttachment:goldenAttachment];
+  } else {
+    NSLog(@"This test will fail - no golden named %@ found. Follow the steps in the README to add "
```

Comment:
XCTFail?

Reply:
I want it to continue so it will generate the potentially good screenshot for you

Reply:
I guess I could generate the good screen shot, attach it, and fail though - which would fail slighty faster than I currently am.  WDYT?

Reply:
It would be better for the test failure to suggest what to do than have to dig through logs.